ocs_ci/helpers/osd_resize.py

```diff
@@ -192,20 +192,40 @@ def check_storage_size_is_reflected(expected_storage_size):
             f"The PV sizes are not equal to the expected storage size {expected_storage_size_in_gb}"
         )
 
+    if expected_ceph_capacity:
+        check_ceph_capacity_increased(expected_ceph_capacity)
+
+
+def check_ceph_capacity_increased(
```

Comment:
do we have validation that all OSDs actually resized successfully before capacity check?

Reply:
Yes, we get the osd size from the storage cluster. See the line https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/helpers/osd_resize.py#L162